chapter_4/domain_and_namespaces.md

```diff
@@ -1,8 +1,76 @@
-# Domain and Namespaces
-
-- Domains
-- Enabling domains
-- Domain priority
-- Import and Export domains
-- cfme-rhconsulting-scripts-> miqimport and miqexport command (https://github.com/rhtconsulting/cfme-rhconsulting-scripts)
-- Namespaces
+## Domain
+
+A domain is a collection of namespaces, classes, instances and methods.
+ManageIQ ship with single domain called ManageIQ. It is in locked state
```

Comment:
Can you please make it more informative "ManageIQ ship with single domain called ManageIQ"